tests/test_transcribe.py

```diff
@@ -99,23 +100,51 @@ def test_smooth_notes(load_sample_audio):
 
     f0_range = (24, 48)
 
+    print("***")
```

Comment:
Generally you don't want to leave `print` statements in your code - they litter. Log debug is a good alternative if you need to have some sort of output.